ivy/array/activations.py

```diff
@@ -1,8 +1,34 @@
 # global
 import abc
+from typing import Optional
 
-# ToDo: implement all methods here as public instance methods
+# local
+import ivy
 
 
 class ArrayWithActivations(abc.ABC):
-    pass
+    def relu(self: ivy.Array, out: Optional[ivy.Array] = None) -> ivy.Array:
+        return ivy.relu(self, out=out)
+
+    def leaky_relu(
+        self: ivy.Array, alpha: Optional[float] = 0.2, out: Optional[ivy.Array] = None
+    ) -> ivy.Array:
+        return ivy.leaky_relu(self, alpha, out=out)
```

Comment:
we actually now pass `self._data` into the functions rather than `self`, small detail, but better to be consistent. Please change for all functions.

Reply:
Hey @kirodev, can you let me know after you are done adding these?